torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1356,6 +1359,35 @@ def _free_full_params_and_use_local_shard(params_to_free):
                         _free_full_params_and_use_local_shard(currently_local_params)
                         self.training_state = TrainingState_.IDLE
 
+    def named_parameters(
+        self,
+        prefix: str = "",
```

Comment:
Let's take in *args, **kwargs so we don't have to change if the API changes?

Reply:
Would it be correct to say that PyTorch Core cannot add positional arguments before `prefix` without breaking backward compatibility? In that case, could we only add `**kwargs`?

Reply:
Using `(*args, **kwargs)` is kind of the convention in PyThon to deal with this case.

Reply:
Oh, I misunderstood. I thought we were trying to _additionally_ pass in `*args, **kwargs` instead of replacing `prefix` and `recurse`.

Fixed this now.